Zapper-fi/studio
# feat(swell): added Swell app and swETH token fetcher

## Description

* Added Swell app and swETH token fetcher

App: https://app.swellnetwork.io/

## Checklist


## How to test?

Run:

`ENABLED_APPS=swell pnpm studio dev`

Visit http://localhost:5001/apps/swell/tokens?groupIds[]=sweth&network=ethereum

Observe fetched token data for swETH repricing token
